subsys/debug/Kconfig

```diff
@@ -261,6 +261,14 @@ config SPIN_VALIDATE
 	  enabled. It adds a relatively hefty overhead (about 3k or so) to
 	  kernel code size, don't use on platforms known to be small.
 
+config SPIN_LOCK_TIME_LIMIT
+	int "Spin lock holding time limit in microseconds"
```

Comment:
Microseconds seems too coarse to me.  Code that lives in this world tends to think a 10us delay is just barely tolerable.